packages/x-date-pickers/src/StaticTimePicker/StaticTimePicker.test.tsx

```diff
@@ -9,12 +9,8 @@ import {
   getAllByRole,
   fireEvent,
 } from '@mui/monorepo/test/utils';
-import { StaticTimePicker } from './StaticTimePicker';
-import {
-  adapterToUse,
-  wrapPickerMount,
-  createPickerRenderer,
-} from '../../../../test/utils/pickers-utils';
+import { adapterToUse, wrapPickerMount, createPickerRenderer } from 'test/utils/pickers-utils';
+import { StaticTimePicker } from '@mui/x-date-pickers/StaticTimePicker';
```

Comment:
I have done this diff

```diff
-import { StaticTimePicker } from './StaticTimePicker';
+import { StaticTimePicker } from '@mui/x-date-pickers/StaticTimePicker';
```

So the tests are more end-to-end (testing the index export too)